SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -1002,15 +1002,24 @@ void FSpatialGDKEditorToolbarModule::NoAutomaticConnectionClicked() const
 {
 	USpatialGDKEditorSettings* SpatialGDKEditorSettings = GetMutableDefault<USpatialGDKEditorSettings>();
 	SpatialGDKEditorSettings->SpatialOSNetFlowType = ESpatialOSNetFlow::NoAutomaticConnection;
+	SpatialGDKEditorSettings->bUseDevelopmentAuthenticationFlow = false;
 	// Unbind the TryStartSpatialDeployment if not in local deployment mode.
 	UEditorEngine::TryStartSpatialDeployment.Unbind();
 	SpatialGDKEditorSettings->SaveConfig();
+
+	// Ensure we disable bUseDevelopmentAuthenticationFlow when we don't connect to spatial deployment.
+	USpatialGDKSettings* SpatialGDKSettings = GetMutableDefault<USpatialGDKSettings>();
+	SpatialGDKSettings->bUseDevelopmentAuthenticationFlow = false;
+	SpatialGDKSettings->SaveConfig();
```

Comment:
The dev auth related fields in the Spatial GDK runtime settings aren't actually saved in the config (only the ones from the editor are).
```suggestion
```